.devops/deploy-pipelines.yml

```
@@ -0,0 +1,123 @@
+# Azure DevOps pipeline to release a new version and deploy to production.
+
+variables:
+  NODE_VERSION: '12.19.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+parameters:
+  - name: 'RELEASE_SEMVER'
+    displayName: 'When packing a release, define the version bump to apply'
+    type: string
+    values:
+      - major
+      - minor
+      - patch
+    default: patch
+
+# Only manual activations are intended
+trigger: none
+pr: none
+
+
+resources:
+  repositories:
+    - repository: pagopaCommons
+      type: github
+      name: pagopa/azure-pipeline-templates
+      ref: refs/tags/v4
+      endpoint: 'io-azure-devops-github-ro'
+
+stages:
+  # Create a release
+  # Activated when ONE OF these are met:
+  # - is on branch master
+  # - is a tag in the form v{version}-RELEASE
+  - stage: Release
+    # condition:
+    #   and(
+    #     succeeded(),
+    #     or(
+    #       eq(variables['Build.SourceBranch'], 'refs/heads/master'),
+    #       and(
+    #         startsWith(variables['Build.SourceBranch'], 'refs/tags'),
+    #         endsWith(variables['Build.SourceBranch'], '-RELEASE')
+    #       )
+    #     )
+    #   )
```

```suggestion
    # condition:
    #   and(
    #     succeeded(),
    #     or(
    #       eq(variables['Build.SourceBranch'], 'refs/heads/master'),
    #       and(
    #         startsWith(variables['Build.SourceBranch'], 'refs/tags'),
    #         endsWith(variables['Build.SourceBranch'], '-RELEASE')
    #       )
    #     )
    #   )
```

use variable DEFAULT_BRANCH instead of branch main/master
DEFAULT_BRANCH is already setted as variable
u can copy from this: https://github.com/pagopa/hub-spid-login-ms/pull/4/files